homework/homework111223.py

```diff
@@ -23,29 +41,29 @@
 Tossing their heads in sprightly dance.
 """
 
-poem_text.lower()
+poem_text = poem_text.lower()
 vowels = ['a', 'e', 'i', 'o', 'u']  # list of vowels
 
 separator_number = 18
 logging.info('-' * separator_number)
 logging.info(f"| {'Vowel':^2} | {'Count':^6} |")
 logging.info('-' * separator_number)
 
-# Counting the number of vowels in a text
+# Task 1. Counting the number of vowels in a text
 for vowel in vowels:
     count = poem_text.count(vowel)
     logging.info(f'| {vowel:^6}|  {count:^5} |')
 
 logging.info('-' * separator_number)
 logging.info('The table output is complete.')
 
-# The second task is to change all the vowels to those provided by the teacher.
-modified_poem_text = (
-    poem_text
-    .replace('A', 'À').replace('a', 'à')
-    .replace('E', 'É').replace('e', 'é')
-    .replace('I', 'Í').replace('i', 'í')
-    .replace('O', 'Ó').replace('o', 'ó')
-    .replace('U', 'Ú').replace('u', 'ú')
-)
+# Task 2. Changing all the vowels to those provided by the teacher.
+new_symbols_for_replacement = str.maketrans({
+    'A': 'À', 'a': 'à',
+    'E': 'É', 'e': 'é',
+    'I': 'Í', 'i': 'í',
+    'O': 'Ó', 'o': 'ó',
+    'U': 'Ú', 'u': 'ú',
+})
+modified_poem_text = poem_text.translate(new_symbols_for_replacement)
```

Comment:
Final poem ( replaced text)  in you case will be without capital letters. Try to understand where in your code fix required

Reply:
Yeah, you're right. I see